server/services/v1/database/query_runner.go

```diff
@@ -324,9 +324,25 @@ func (runner *DeleteQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 
 	ts := internal.NewTimestamp()
 
+	var reqStatusFound bool
+	var reqStatus *metrics.RequestStatus
+
+	if config.DefaultConfig.GlobalStatus.Enabled {
+		reqStatus, reqStatusFound = metrics.RequestStatusFromContext(ctx)
```

Comment:
same as above, just call `metrics.RequestStatusFromContext(ctx)`, and as it will return notFound then the remaining code will remain the same.

Reply:
I changed this.